apps/root/src/state/balances/actions.ts

```diff
@@ -76,37 +77,41 @@ export const fetchInitialBalances = createAppAsyncThunk<
   const chainIds = Object.keys(tokenListByChainId).map((chainId) => Number(chainId));
   const wallets = accountService.getWallets().map((wallet) => wallet.address);
 
-  const accountBalancesResponse = await meanApiService.getAccountBalances({
-    wallets,
-    chainIds,
-  });
   const parsedAccountBalances: Omit<BalancesState, 'isLoadingAllBalances'> = {};
 
-  for (const [walletAddress, chainBalances] of Object.entries(accountBalancesResponse.balances)) {
-    for (const [chainIdString, tokenBalance] of Object.entries(chainBalances)) {
-      const chainId = Number(chainIdString);
-
-      for (const [tokenAddress, balance] of Object.entries(tokenBalance)) {
-        const token = unwrapResult(
-          await dispatch(
-            fetchTokenDetails({
-              tokenAddress,
-              chainId: chainId,
-              tokenList: tokenListByChainId[chainId],
-            })
-          )
-        );
-
-        set(
-          parsedAccountBalances,
-          [chainId, 'balancesAndPrices', tokenAddress, 'balances', walletAddress],
-          BigInt(balance)
-        );
-        set(parsedAccountBalances, [chainId, 'balancesAndPrices', tokenAddress, 'token'], token);
+  try {
```

Comment:
No need to wrap this, since the actions already provide a `.rejected` value just like we use the `success` and `pending`